sync.sh

```diff
@@ -10,18 +17,32 @@
 if [ "${DATABASE:=redis}" = redis ]; then
      PERSIST="-redis"
 else
-     PERSIST="-mongo"
+     PERSIST=${MONGO_PERSIST}
 fi
 
 # nightly or other release
 USE_RELEASE=${RELEASE:-nightly-build}
 if [ "$USE_RELEASE" = "nightly-build" ]; then
-     COMPOSE_FILE="docker-compose-nexus${PERSIST}${USE_NO_SECURITY}${USE_ARM64}.yml"
-     wget -q -O ${COMPOSE_FILE} "https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/nightly-build/compose-files/${COMPOSE_FILE}"
+  if [ "$PERSIST" = ${MONGO_PERSIST} ]; then
+     echo "Mongo no longer supported. No compose file(s) downloaded"
+     exit
+  fi
+
+   # Download all the files, even if may not be used, so 'down' target works.
+   wget -q ${WINDOWS_WGET_OPTION} -O Makefile \
+      "${NIGHT_BUILD_URL}/Makefile"
+   wget -q ${WINDOWS_WGET_OPTION} -O docker-compose-portainer.yml \
+      "${NIGHT_BUILD_URL}/docker-compose-portainer.yml"
+   wget -q ${WINDOWS_WGET_OPTION} -O docker-compose-nexus-base.yml \
+      "${NIGHT_BUILD_URL}/docker-compose-nexus-base.yml"
+   wget -q ${WINDOWS_WGET_OPTION} -O docker-compose-nexus-add-security.yml \
+      "${NIGHT_BUILD_URL}/docker-compose-nexus-add-security.yml"
+   wget -q ${WINDOWS_WGET_OPTION} -O docker-compose-nexus-add-mqtt.yml \
+      "${NIGHT_BUILD_URL}/docker-compose-nexus-add-mqtt.yml"
+   wget -q ${WINDOWS_WGET_OPTION} -O docker-compose-nexus-ui.yml \
+      "${NIGHT_BUILD_URL}/docker-compose-nexus-add-mqtt.yml"
```

Comment:
The filename of docker-compose-nexus-add-mqtt.yml is duplicated.

Reply:
Good catch! Fixed, THX!